src/Select.tsx

```diff
@@ -14,10 +23,18 @@ export type SelectProps = Partial<Omit<_SelectProps, 'children' | 'renderer'>> &
   }>;
 
 function Select(props: SelectProps, ref: ForwardedRef<SelectElement>): ReactElement | null {
+  const innerRef = useRef<SelectElement>(null);
```

Comment:
Could we reuse the ref that is passed down to this component?

Reply:
Unfortunately, no, because we have no idea if the user provided a function or a `MutableRefObject`.